lib/Peep.rb

```diff
@@ -0,0 +1,14 @@
+require 'pg'
+
+class Peep
+  def self.all
+    if ENV['Environment'] == 'test'
```

Comment:
Might be better if `ENVIRONMENT` is in all caps, which helps to clearly indicate that it's an environment variable